src/main/java/uk/gov/ons/census/casesvc/service/QuestionnaireLinkedService.java

```diff
@@ -110,11 +109,8 @@ private void logUnlinkedCaseEventAgainstOriginalCaseIfQidIsNowLinkedToNewCase(
     }
   }
 
-  private boolean checkRequiredFieldsForIndividualHI(
-      String questionnaireId, String caseType, String individualCaseId) {
-    return (isIndividualQuestionnaireType(questionnaireId)
-        && caseType.equals("HH")
-        && individualCaseId != null);
+  private boolean checkRequiredFieldsForIndividualHI(String questionnaireId, String caseType) {
+    return (isIndividualQuestionnaireType(questionnaireId) && caseType.equals("HH"));
   }
 
   private void throwIllegalArgumentExceptionIfIndQIDAndIndCaseIdPresentButCaseTypeNotHH(
```

Comment:
While you're tidying things up, you could rename this ridiculous function

Reply:
@NickGrantONS Renamed it to something a little less extravagantly long (but hopefully still descriptive enough).